mypy/exprtotype.py

```diff
@@ -43,7 +43,7 @@ def expr_to_unanalyzed_type(expr: Expression) -> Type:
     elif isinstance(expr, ListExpr):
         return TypeList([expr_to_unanalyzed_type(t) for t in expr.items],
                         line=expr.line, column=expr.column)
-    elif isinstance(expr, (StrExpr, BytesExpr)):
+    elif isinstance(expr, (StrExpr, BytesExpr, UnicodeExpr)):
```

Comment:
From a quick grep it looks like there might be -- I'll take a closer look tomorrow!